paper/paper.md

```diff
@@ -0,0 +1,457 @@
+higher quality content. <!-- TODO: cite U-M Carpentries paper for the development model once it's submitted to bioRxiv?-->
+While teaching from the curriculum at the first Data Science Summer Experience,
+instructors took notes on their experience and made revisions afterward.
+Maintainers continue to monitor the repository and resolve issues as they arise.
+
+Following the onset of the COVID-19 pandemic, we quickly pivoted our club to a
+virtual format.
+In preparation for the 2020 Summer Experience, we switched to a flipped
+classroom style following feedback from our club participants that it was too
+difficult to follow along live coding via Zoom.
+We recorded facilitators teaching the lesson notebooks as if they were live
+coding, then shared them along with a link to the lesson notebooks for students
+to code along with while watching the videos.
+Each video shows the Jupyter notebook alongside the facilitator themselves
+teaching.
+This format allowed students to learn at their own pace, then ask questions and
+practice when we met together virtually. <!-- TODO: should this actually go later in the curriculum section?-->
+
+Our Jupyter notebooks and links to the lesson videos can be accessed on GitHub:
+https://github.com/GWC-DCMB/curriculum-notebooks
+
+# Curriculum
+
+Our curriculum was designed for high school students with no prior coding
+experience who are interested in learning Python programming for data science.
+However, this course material would be useful for anyone interested in learning
+basic programming for data analysis, regardless of age or experience level.
+
+Our curriculum features short lessons to deliver course material in “bite sized”
+chunks, followed by practices to solidify the learners' understanding.
+
+## Learning Objectives
+
+Our learning objectives are:
+
+1. Understand fundamental concepts and best practices in coding.
+2. Apply data analysis to real world data to answer scientific questions.
+3. Create informative summary statistics and data visualizations in Python.
+
+These skills provide a solid foundation for basic data analysis in Python and
+participation in our events exposes students to the many ways coding and
+data science can be used to make large impacts across many disciplines.
+
+## Course Content
+
+Our curriculum design consists of 27 lessons broken up into 5 modules that cover Jupyter notebook setup,
+Python coding fundamentals, use of essential data science packages including
+Pandas and numpy, basic statistical analyses, and plotting using seaborn and
+matplotlib (Figure 1) [@harris_array_2020; @waskom_seaborn_2021;
+@hunter_matplotlib_2007].
+Each of the following modules consists of a lesson notebook, used for teaching
+the concept via live coding, and a practice notebook containing similar
+exercises for the student to complete on their own following the lesson:
+
+Jupyter Notebook Setup:
+1. Jupyter Setup
+
+Coding Fundamentals:
```

Comment:
Oh no I messed up the number formatting. If anyone knows an easy fix, please help. Otherwise I'll look into it later.

Reply:
Did you though? It looks like someone typed in the raw numbers rather than using markdown's automatic numbering, so they look correct.

Reply:
Oh I see the problem. Needed a newline between the previous "paragraph" (actually the sublist title) and the start of the list.